scripts/code_size_compare.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
+"""
+This file is part of Mbed TLS (https://tls.mbed.org)
+
+Copyright (c) 2021, Arm Limited, All Rights Reserved
+
+Purpose
+
+This script is for comparing the size of the library files from two
+different Git revisions within an Mbed TLS repository.
+The results of the comparison is formatted as csv and stored at a
+configurable location.
+Note: must be run from Mbed TLS root.
+"""
+
+import argparse
+import os
+import subprocess
+
+
+class CodeSizeComparison:
+    """compare code size between two Git revisions"""
```

Comment:
`"""` is multi line string tag.

Reply:
I think triple double quotes are also used to indicate a docstring, so in this case the string would be applied to the `__doc__`  special attribute of the `CodeSizeComparson` class. From a style perspective though I would prefer proper capitalisation and punctuation so `"""Compare code size between two Git revisions."""`.

Reply:
The docstring is the first statement in the class/function/…, no matter what quotes it uses (and it can even use mixed quotes, e.g. `"foo"'bar'` is a valid docstring). The convention ([PEP 257](https://www.python.org/dev/peps/pep-0257/)) is to always use triple-double-quotes for docstrings.